### PR TITLE
Export Tip, AlphaWarning, DeprecatedWarning components

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/3532714909/pulses/4856297572

### CODE REVIEW

File: src/components/tip/tip.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export interface TipProps {
   emoji?: string;
 }
 
-const Tip: React.FC<TipProps> & {
+export const Tip: React.FC<TipProps> & {
```
Comment (talkor) [inline src/components/tip/tip.tsx:21]: It's already exported as default below

Comment (SergeyRoyt) [reply src/components/tip/tip.tsx]: Moved all the components to default exports and removed named exports

File: src/components/index.ts [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
-import { SectionName } from './section-name/section-name';
-import { StoryDescription } from './story-description/story-description';
-// import { Tip } from "./tip/tip"; // uses attention box - not exported for now
-import { Title } from './title/title';
-import { TokenTable } from './token-table/token-table';
-import { UnstyledList } from './unstyled-list/unstyled-list';
-import { UnstyledListItem } from './unstyled-list-item/unstyled-list-item';
-import { UsageGuidelines } from './usage-guidelines/usage-guidelines';
-import { VisualDescription } from './visual-description/visual-description';
+import { default as AnchorListItem } from './anchor-list-item/anchor-list-item';
```
Comment (talkor) [inline src/components/index.ts:1]: I think you can export them already
```suggestion
export { default as AnchorListItem } from './anchor-list-item/anchor-list-item';
```

**Review by talkor — APPROVED** [review]
Amazing 💪🏼 💪🏼

**Pull request merged**